src/components/LoginButton.jsx

```diff
@@ -1,14 +1,9 @@
 import React from "react";
 
-const LoginButton = () => {
+const LoginButton = (className) => {
   return (
     <>
-      <button
-        className="btn middle rounded-[3px] duration-500 ease-in-out center font-medium text-white text-ellipsis shadow-md transition-all hover:shadow-lg border border-collapse border-x-2 border-y-4 border-green-800"
-        data-ripple-light="true"
-      >
-        Login
-      </button>
+      <div className={className}></div>
```

Comment:
you can pass some Title or heading to make it more generic

Reply:
Login Button can now be removed @verma5309 @kanayan007